spec/models/event_procedure_spec.rb

```diff
@@ -54,6 +54,15 @@
     it { is_expected.to validate_presence_of(:patient_service_number) }
     it { is_expected.to validate_presence_of(:room_type) }
     it { is_expected.to validate_presence_of(:payment) }
+
+    context "when validating custom and urgency" do
+      let(:event) { build(:event_procedure, urgency: true, procedure_attributes: { custom: true }) }
+
+      it "is invalid" do
```

Comment:
```suggestion
      it "is invalid when urgency is true and custom is true" do
```
Also add the following scenarios
```
      it "is valid when urgency is true and custom is false" do
      it "is valid when urgency is false and custom is true" do
      it "is valid when urgency is true  and custom is true" do
```

Reply:
Ok